nmstate-varlink/varlink_service_test.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+import libnmstate
+import varlink
+import threading
+
+from nmstate_varlink import ServiceRequestHandler, service
+
+from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceType
+from libnmstate.schema import InterfaceState
+from libnmstate.schema import InterfaceIPv4
+from libnmstate.schema import InterfaceIPv6
+from libnmstate.schema import DNS
+from libnmstate.schema import Route
+from libnmstate.schema import RouteRule
+
+
+address = "unix:/tmp/nmstate"
+# address = "tcp:127.0.0.1:6000"
+
+
+@pytest.fixture()
+def varlink_server():
```

Comment:
Please use run_server from varlink to test the code from the varlink implementation instead of re-writing it here.